Workspaces/Base/WorkspaceUI.cs

```diff
@@ -444,6 +451,185 @@ IEnumerator Start()
 			yield return null;
 		}
 
+		void AdjustHandlesAndIcons()
```

Comment:
I did consider this but nothing that I could think of was any shorter or easier to read. I'd be happy to discuss tomorrow.